### DISCUSSION TITLE
When are Converters' `ConvertBack` method called?

### DISCUSSION THREAD

**Discussion by Abe-Froman** [CommunityToolkit/Windows#301] [Q&A] [answered]
I'm trying to understand why generated code is hitting [this](https://github.com/CommunityToolkit/Windows/blame/8e4bd7a213210feff42ade340a8be6624e9f5c64/components/Converters/src/TypeToObjectConverter.cs#L88C8-L88C8) `NotImplementedException` in `TypeToObjectConverter`. I don't believe I'm calling this in a way contrary to how I'm using it in other locations, just trying to understand what mechanism calls the `ConvertBack` method in the first place.

Edit:
I've now refactored my code away from using the converter causing the issue but will leave the question here as I'd like to learn more about ConvertBack and its use.

**Comment by insomniachi** [comment, marked as answer]
Convert back is called if your binding is two way and the control has changed the bound value.

**Reply by Abe-Froman** [reply]
Thank you @insomniachi
